R/req-perform-stream.R

```diff
@@ -9,8 +9,12 @@
 #' @param callback A single argument callback function. It will be called
 #'   repeatedly with a raw vector whenever there is at least `buffer_kb`
 #'   worth of data to process. It must return `TRUE` to continue streaming.
-#' @param timeout_sec Number of seconds to processs stream for.
+#' @param timeout_sec Number of seconds to process stream for.
 #' @param buffer_kb Buffer size, in kilobytes.
+#' @param round How should the raw vector sent to `callback` be rounded?
+#'   Choose `"byte"`, `"line"`, or supply your own function that takes a
+#'   raw vector of `bytes` and returns the locations of possible cut points
```

Comment:
I think returning a vector of cut points makes the implementation a bit simpler; see the new implementation of `round_lines()` below. What do you think?

Reply:
Thanks for improving the doc. With this reframe, returning the vector of cut points does make more sense, although, the current implementation only cares about the last one.